src/Tizen.NUI/src/public/WindowSystem/TizenShell.cs

```diff
@@ -0,0 +1,132 @@
+﻿/*
+ * Copyright(c) 2020 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Tizen.Applications.Exceptions;
+
+namespace Tizen.NUI.WindowSystem.Shell
+{
+    /// <summary>
+    /// Tizen Window System Shell.
+    /// This is a basic handle class for others in WsShell.
+    /// Others class in WsShell using this basic class to use TIzen Window System Shell.
+    /// </summary>
+    /// <since_tizen> 8 </since_tizen>
+    public class TizenShell : IDisposable
+    {
+        private IntPtr _tzsh;
+        private bool disposed = false;
+
+        /// <summary>
+        /// Throw error codes generated by tizen ws shell
+        /// </summary>
+        /// <param name="error"></param>
+        /// <since_tizen> 8 </since_tizen>
+        public void ErrorCodeThrow(int error)
+        {
+            if (error == (int)Interop.TizenShell.ErrorCode.None)
+            {
+                ;
+            }
```

Comment:
Do you really need these lines?
Compiler may remove it but you don't have to create a situation that is not sure.

Reply:
If this package works normally, this library get only 5 types of return value.
(None, OutOfMemory, InvalidParameter, PermissionDenied, NotSupported, NoService)
But this APIs get return value as Server, sometimes we can get unexpected return value.
So we handle all of  return type error in ErrorCodeThrow().
if except this, we need to add more codes to use ErrorCodeThrow().

For example.. currently
```
_res = tzshQuickPanelScrollableStateGet(_tzshQpClient, out _scroll);
_tzsh.ErrorCodeThrow(_res);
return (_res);
```

but if we remove that unmeaningful codes, we need to add below condition before use ErrorCodeThrow()
```
_res = tzshQuickPanelScrollableStateGet(_tzshQpClient, out _scroll);
if (_res != (int)Interop.TizenShell.ErrorCode.None)
    _tzsh.ErrorCodeThrow(_res);
return (_res);
```

Reply:
Just finish the method with return statement if you want ignore ErrorCode.None.
```
if (error == (int)Interop.TizenShell.ErrorCode.None)
{
    return;
}
```

Reply:
I fixed it.